app/assets/javascripts/tutorials.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
In general, I like to delete files I'm not using that Rails auto creates. It can become hard to know what is important or not later down the line, and it ends up cluttering a bit. I remember seeing a tweet once that this one Rails programmer looks for the existence of an empty concerns folder when evaluating candidate code (keeping the folder if they're not using it), because they didn't pay attention to the small details